CHANGELOG.md

```diff
@@ -6,7 +6,52 @@ This projects adheres to [Semantic Versioning](http://semver.org/) and [Keep a C
 ## [Unreleased]
 
 ### Added
+- `WordPress.CSRF.NonceVerification` sniff to flag form processing without nonce verification.
+
+## [0.4.0] - 2015-5-1
+
+### Added
 - Change log file.
+- Handling for string-interpolated input variables in the `WordPress.VIP.ValidatedSanitizedInput` sniff.
+- Errors for using uncached functions when cached equivalents exist.
+- `space_before_colon` setting for the `WordPress.WhiteSpace.ControlStructureSpacing` sniff, for control structures using alternative syntax. Possible values: `'required'`, `'optional'`, `'forbidden'`.
+- Support for `sanitization` whitelisting comments for the `WordPress.VIP.ValidatedSanitizedInput` sniff.
+- Granular error/warning names for all errors and warnings.
+- Handling for ternary conditions in the `WordPress.XSS.EscapeOutput` sniff.
+- `die`, `exit`, `printf`, `vprintf`, `wp_die`, `_deprecated_argument`, `_deprecated_function`, `_deprecated_file`, `_doing_it_wrong`, `trigger_error`, and `user_error` to the list of printing functions in the `WordPress.XSS.EscapeOutput` sniff.
+- `customPrintingFunctions` setting for the `WordPress.XSS.EscapeOutput` sniff.
+- `rawurlencode()` and `wp_parse_id_list()` to the list of "sanitizing" functions in the `WordPress.XSS.EscapeOutput` sniff.
+- `json_encode()` to the list of discouraged functions in the `WordPress.PHP.DiscouragedFunctions` sniff, in favor of `wp_json_encode()`.
+- `vip_powered_wpcom()` to the list of auto-escaped functions in the `WordPress.XSS.EscapeOutput` sniff.
+- `debug_print_backtrace()` and `var_export()` to the list of discouraged functions in the `WordPress.PHP.DiscouragedFunctions` sniff.
+- Smart handling for formatting functions (`sprintf()` and `wp_sprintf()`) in the `WordPress.XSS.EscapeOutput` sniff.
+- `WordPress.PHP.StrictComparisons` sniff.
+- Correct handling of `array_map()` in the `WordPress.VIP.ValidatedSanitizedInput` sniff.
+- `$_COOKIE` and `$_FILE` to the list of superglobals flagged by the `WordPress.VIP.ValidatedSanitizedInput` and `WordPress.VIP.SuperGlobalInputUsage` sniffs.
+- `$_SERVER` to the list of superglobals flagged by the `WordPress.VIP.SuperGlobalInputUsage` sniff.
+- `Squiz.ControlStructures.ControlSignature` sniff to the rulesets.
+
+### Changed
+- `WordPress.Arrays.ArrayKeySpacingRestrictions` sniff to give errors for `NoSpacesAroundArrayKeys` and `SpacesAroundArrayKeys` instead of just warnings.
+- `WordPress.NamingConventions.ValidFunctionName` sniff to allow for camel caps method names in child classes.
+- `WordPress.XSS.EscapeOutput` sniff to allow for integers (e.g. `echo 5` and `print( -1 )`).
+
+### Removed
+- Errors for mixed key/keyless array elements in the `WordPress.Arrays.ArrayDeclaration` sniff.
+- BOM from `WordPress.WhiteSpace/OperatorSpacing` sniff file.
```

Comment:
Is this `/` supposed to be a `.`, or is the `.` supposed to be a `/`, or neither?